### DISCUSSION TITLE
drug product question

### DISCUSSION THREAD

**Discussion by masciam** [mcwdsi/dron#78] [General]
Drug product is defined in DRON as the following: a material entity (1) containing at least one scattered molecular aggregate as part that is the bearer of an active ingredient role and (2) that is itself the bearer of a clinical drug role. http://purl.obolibrary.org/obo/DRON_00000005
scattered molecular aggregate is defined in DROn as the following: a material entity that consists of all the molecules of a specific type that are located in some bounded region and which is part of a more massive material entity that has parts that are other such aggregates.
http://purl.obolibrary.org/obo/OBI_0000576
My question is:
I need to have as a subgroup of drug product drugs that are not only chemicals but could be antibodies or cell Does the scattered molecular aggregates includes those? or as I intended is only referring to chemicals?
Thanks
AM

**Comment by dillerm** [comment]
@masciam , antibodies should be fine, as ChEBI represents things like proteins as molecular entities. However, I'm not sure about 'cell'. Could you elaborate on that?

**Comment by masciam** [comment]
Thanks Matthew,
I was thinking about the cell used as cancer therapy I think those should
still be classified as drug.
I have another question for you:
I am using Dron to create an application ontology for MD Anderson and I
have use the previous version of Dron and I figure out that a lot of drugs
are not listed and that the mapping to RxNorm is not consistent with the
actual RxNorm ID.
Are you envisioning to update that and the list of drugs?
Thank you
AM

**Comment by hoganwr** [comment]
It would be helpful to know which version of DrOn and to have examples of "not consistent" RxNorm IDs and missing things. We do our best to keep consistent with RxNorm. We can work on fixing any issues with the RxNorm conversion process.

For example, newer versions of DrOn created by the new "build from templates" process have thousands of Semantic Branded Drugs included that were missing from prior versions built by the old process.

Conversion to the new process also found smaller numbers (<100 typically) of other term types missing.

**Comment by masciam** [comment]
I have used the June 2020 version.
 For that reason I I wanted to ask if the new version has been updated.
Thanks
AM

**Comment by hoganwr** [comment]
We discussed this issue at the July 2025 DrOn meeting. If there are drug products in RxNorm that contain cells, then the current definitions of 'drug product' and 'clinical drug product' would need to be updated. One easy fix, if required is to say that a drug product contains either a scattered molecular aggregate or an aggregate of cells. However, proper modeling of cell-containing products would then require a significant change to the build process from RxNorm. The next task is to see what if any cellular products are there. A search for "CART" and "CAR-T" did not return anything.

**Reply by hoganwr** [reply]
And yes, even if not needed for RxNorm, AM, we could still update the definition.

**Reply by hoganwr** [reply]
I can confirm that there are multiple drug products with cells. There are a number of injectable products with units of "cells/ML". That's also how we could identify them if we choose to represent them differently from others.

**Reply by masciam** [reply]
Ok great!

**Comment by hoganwr** [comment]
We requested a new class from OBI to make this possible.

See: https://github.com/obi-ontology/obi/issues/1977
